src/TraceEvent/Symbols/SymbolReader.cs

```diff
@@ -880,8 +880,13 @@ internal bool GetPhysicalFileFromServer(string serverPath, string pdbIndexPath,
                             m_log.WriteLine("FindSymbolFilePath: In task, sending HTTP request {0}", fullUri);
 
                             var req = (System.Net.HttpWebRequest)System.Net.HttpWebRequest.Create(fullUri);
+#if ! NETSTANDARD1_6
                             req.UserAgent = "Microsoft-Symbol-Server/6.13.0009.1140";
```

Comment:
Don't bother making this 1.6 specific.  I am OK with it being unconditional.  I only put that in because I seemed to need it to make it work, but that does not seem to be the case anymore.